tools/meson.build

```diff
@@ -5,12 +5,14 @@ bubblejail_test_cmd_run = run_target(
     depends : bubblejail_lib_package,
     env : test_env,
 )
-bubblejail_test_gui_run = run_target(
-    'bubblejail_test_gui_run',
-    command : [py_installation, files('run_test_bubblejail.py'), 'gui'],
-    depends : bubblejail_lib_package,
-    env : test_env,
-)
+if get_option('gui').enabled()
```

Comment:
Is it really needed to disable this run target? It does not produce any build artifacts.

Reply:
fixed